packages/devtools-plugin/src/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { createPlugin } from "@prodo/core/lib/plugins";
+import { Event } from "@prodo/core/lib/types";
+import DevTools, { DevMessage } from "@prodo/devtools-core";
+import { original } from "immer";
+import * as _ from "lodash";
+
+// tslint:disable-next-line:no-empty-interface
+export interface DevToolsConfig {}
```

Comment:
This seems unnecessary. Can just use `{}` below

Reply:
Yeah, I think was half-adding an option to the config to turn the devtools on and off and then didn't fully clean it up - might use it for that (or get rid of it)